README.md

```diff
@@ -23,10 +25,21 @@ Now any templates ending in .emblem will be compiled as Ember-Handlebars templat
 You can overwrite config as the followings:
 
 ``` ruby
-Ember::Emblem::Template.configure do |config|
-  config.precompile = true
-
-  # You can overwrite other config
+if defined?(Ember::Emblem::Template)
```

Comment:
Ah, makes sense. Should probably change this to use the Rails configuration
system at some point.

On Wed, Sep 23, 2015, 10:36 AM Benjamin ter Kuile <notifications@github.com>
wrote:

> In README.md
> <https://github.com/botandrose/ember-emblem-template/pull/1#discussion_r40232967>
> :
>
> > @@ -23,10 +25,21 @@ Now any templates ending in .emblem will be compiled as Ember-Handlebars templat
> >  You can overwrite config as the followings:
> >
> >  ``` ruby
> > -Ember::Emblem::Template.configure do |config|
> > -  config.precompile = true
> > -
> > -  # You can overwrite other config
> > +if defined?(Ember::Emblem::Template)
>
> This code most likely will end up in a rails initializer. Since not
> loading the full asset stack in production because they all are precompiled
> is a good thing I think, using it without the defined? check would crash
> the application in production since the gems will not be loaded.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/botandrose/ember-emblem-template/pull/1/files#r40232967>
> .
>